common/views/components/PageLayout/PageLayout.tsx

```diff
@@ -69,6 +70,11 @@ const PageLayoutComponent: FunctionComponent<ComponentProps> = ({
   hideFooter = false,
   excludeRoleMain = false,
 }: ComponentProps) => {
+  const hotjarUrls = ['YSZh_hEAAGTR0xn7', 'YRU7vBEAAOvZyGS2']; // Photo stories
```

Comment:
I would still like a better way to be able to turn hotjar on and off for particular parts of the site and/or specific pages, ideally without having to deploy as part of making it happen. Possibly a Primsic singleton type that is just a list of strings for matching here?

Reply:
Maybe we can cover this when we have our Prismic chat today.

Reply:
There might also be controls in hotjar itself, I was having a poke around a while back but it wasn't a priority. If we could do it in there, it might be simpler.

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/issues/6907

Reply:
I looked into this a bit, too, but I don't _believe_ it's something that's feasible from within hotjar. I'd like to bring #6907 to planning at some point soon, just to make this a nicer experience for anyone who wants to add hotjar tracking (@alicerichmond and @taceybadgerbrook predominantly)

Reply:
Bring it on Tuesday.